aws/emr/cdo-logs/script/elb_mapper.rb

```diff
@@ -0,0 +1,11 @@
+ARGF.each do |line|
+  # Determine whether the log is a GET or POST request, emitting the appropriate
+  # key/value.
+  if line.include? "GET"
+    puts "GET" + "\t" + "1"
```

Comment:
Would this be appreciably slower if you used JSON instead of tab-separated values?  That seems easier than inventing our own tab-separated encoding format.